### PR TITLE
Cr51_remains

### PR DESCRIPTION
**Pull request by ue71603**
replaces https://github.com/NeTEx-CEN/NeTEx/pull/51/files. All that is needed

As Stefan mentionend: No documentation so far. So I added Christophes label here as well.

### CODE REVIEW

File: xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd [hunk 1]
```diff
@@ -654,7 +654,7 @@ Rail transport, Roads and Road transport
 				<xsd:sequence>
 					<xsd:group ref="VehicleJourneyStopAssignmentGroup">
 						<xsd:annotation>
-							<xsd:documentation>Elements for a VEHICLE JOURNEY STOP ASSIGNMENT.  Original Passenger STop Assignment for which this is a change.</xsd:documentation>
+							<xsd:documentation>Elements for a VEHICLE JOURNEY STOP ASSIGNMENT.  Original Passenger Stop Assignment for which this is a change.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:657]:
```suggestion
							<xsd:documentation>Elements for a VEHICLE JOURNEY STOP ASSIGNMENT.  Original PASSENGER STOP ASSIGNMENT for which this is a change.</xsd:documentation>
```

File: xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd [hunk 2]
```diff
@@ -670,5 +670,23 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="PassengerStopAssignmentRef" minOccurs="0"/>
 		</xsd:sequence>
 	</xsd:group>
+	
+		<xsd:complexType name="VehicleJourneyStopAssignmentsInFrame_RelStructure">
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:674]:
```suggestion
	<xsd:complexType name="VehicleJourneyStopAssignmentsInFrame_RelStructure">
```

File: xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd [hunk 3]
```diff
@@ -670,5 +670,23 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="PassengerStopAssignmentRef" minOccurs="0"/>
 		</xsd:sequence>
 	</xsd:group>
+
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:673]:
```suggestion
```

Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:673]:
```suggestion
```

**Review by klement-MENTZ — APPROVED** [review]

File: xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd [hunk 4]
```diff
@@ -670,5 +670,22 @@ Rail transport, Roads and Road transport
...
+					<xsd:element ref="VehicleJourneyStopAssignment" maxOccurs="unbounded">
+						<xsd:annotation>
+							<xsd:documentation>The allocation of a SCHEDULED STOP POINT (i.e. a SCHEDULED STOP POINT of a SERVICE PATTERN or JOURNEY PATTERN) to a specific STOP PLACE or QUAY, for either a Passenger JOURNEY or VEHICLE SERVICE.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>	
+
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:689]:
```suggestion
```

**Review by skinkie — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

**Comment by Aurige** [comment]
No need for documentation (XXXinFrame and associated relations are systematic and not explicitly documented)

**Pull request merged**